src/Core/Command/UploadAvatarHandler.php

```diff
@@ -96,7 +96,10 @@ public function handle(UploadAvatar $command)
         $this->validator->assertValid(['avatar' => $file]);
 
         $manager = new ImageManager;
-        $manager->make($tmpFile)->fit(100, 100)->save();
+
+        $encodedImage = $manager->make($tmpFile)->fit(100, 100)->encode('jpg', 100);
+
+        @file_put_contents($tmpFile, $encodedImage);
```

Comment:
Thanks for this PR. Just two questions;

- using `@` will silence any errors, thus you'll never know whether saving the temporary file succeeded.
- why did you fall back on `file_put_contents`, the manager used `save()` previously, can't we keep using that?